commit a41b62bd85688bec566a7760cea95341bcb7a953
Author: Kefu Chai <kchai@redhat.com>
Date:   Sun Jun 27 12:01:17 2021 +0800

    mgr,mon: s/boost::optional/std::optional/
    
    since the encoding schemes of boost::optional and std::optional are
    identical, they can be used interchangeably.
    
    Signed-off-by: Kefu Chai <kchai@redhat.com>

diff --git a/src/mgr/PyModule.cc b/src/mgr/PyModule.cc
index c554ff1d00a..144f4354c2f 100644
--- a/src/mgr/PyModule.cc
+++ b/src/mgr/PyModule.cc
@@ -185,7 +185,7 @@ PyModuleConfig::~PyModuleConfig() = default;
 std::pair<int, std::string> PyModuleConfig::set_config(
     MonClient *monc,
     const std::string &module_name,
-    const std::string &key, const boost::optional<std::string>& val)
+    const std::string &key, const std::optional<std::string>& val)
 {
   const std::string global_key = "mgr/" + module_name + "/" + key;
   Command set_cmd;
diff --git a/src/mgr/PyModule.h b/src/mgr/PyModule.h
index 037a7a22b00..fbb56080efa 100644
--- a/src/mgr/PyModule.h
+++ b/src/mgr/PyModule.h
@@ -181,6 +181,6 @@ public:
   std::pair<int, std::string> set_config(
     MonClient *monc,
     const std::string &module_name,
-    const std::string &key, const boost::optional<std::string>& val);
+    const std::string &key, const std::optional<std::string>& val);
 
 };
